pkg/controller/datavolume-controller.go

```diff
@@ -682,6 +686,10 @@ func updateProgressUsingPod(dataVolumeCopy *cdiv1.DataVolume, pod *corev1.Pod) e
 	return err
 }
 
+func errConnectionRefused(err error) bool {
+	return strings.Contains(err.Error(), "connection refused")
```

Comment:
Spent a bit of time looking for a better way to check this specific error but I think this is the way to go.